tough/Cargo.toml

```diff
@@ -17,10 +18,13 @@ olpc-cjson = { version = "0.1.0", path = "../olpc-cjson" }
 pem = "0.8.1"
 reqwest = { version = "0.10.4", optional = true, default-features = false, features = ["blocking"] }
 ring = { version = "0.16.13", features = ["std"] }
+rusoto_core = { version = "0.44", optional = true, default-features = false }
+rusoto_kms = { version = "0.44", optional = true, default-features = false }
 serde = { version = "1.0.110", features = ["derive"] }
 serde_json = "1.0.53"
 serde_plain = "0.3.0"
 snafu = "0.6.8"
+tokio = "0.2.13"
```

Comment:
This should also be a transient dependency, not needed directly by `tough`. (i.e. needed by tough-ssm, but not in tough's cargo.toml)

```suggestion
```